src/gatsby-node.js

```diff
@@ -1,47 +1,66 @@
-const googleapi = require(`./googleapis`);
-const path = require(`path`);
-const mkdirp = require(`mkdirp`);
-const fs = require(`fs`);
+const googleapi = require(`./googleapis`)
```

Comment:
Again, lots of small, unnecessary formatting changes by prettier. I'll set up a prettierrc soon, but could you again only change what's actually needed?